pkg/engine/engine.go

```diff
@@ -433,64 +517,46 @@ type detectableChunk struct {
 func (e *Engine) detectorWorker(ctx context.Context) {
 	var wgDetect sync.WaitGroup
 
+	// Reuse the same map to avoid allocations.
+	const avgDetectorsPerChunk = 2
+	uniqueDetectors := make(map[detectorspb.DetectorType]detectorInfo, avgDetectorsPerChunk)
```

Comment:
To reprise our previous conversation, I still find this identifier confusing as a reader. What about `chunkSpecificDetectors` or `detectorsForThisChunk`?